twitter.html

```diff
@@ -0,0 +1,227 @@
+    <header class="nav-bar three-row-grid">
+            <div class="logo">
+                <img class="twitter-logo" src="/web-images/twitter-logo.png" alt="twitterlogo">
+            </div>
+            <nav>
+                <div class="icons">
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/home.png" alt="home">
+                   <a href="#"><h2 class="nav-headers">Home</h2> </a>    
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/hashtag.png" alt="hashtag">
+                    <a href="#"><h2 class="nav-headers">Explore</h2> </a>
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/notifications.png" alt="notifications">
+                    <a href="#"><h2 class="nav-headers">Notifications</h2></a>
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/message.png" alt="message">
+                    <a href="#"><h2 class="nav-headers">Messages</h2></a>
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/bookmark.png" alt="bookmark">
+                    <a href="#"><h2 class="nav-headers">bookmark</h2></a>
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/lists.png" alt="lists">
+                    <a href="#"><h2 class="nav-headers">Lists</h2></a>
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/profile.png" alt="profile">
+                   <a href="#"><h2 class="nav-headers">Profile</h2></a>
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/more.png" alt="more">
+                    <a href="#"><h2 class="nav-headers">More</h2></a>
+                </div>
+                <div class="tweet">
+                    <h3>Tweet</h3>
+                </div>
+            </nav>
+            <div class="profile">
+                <div class="three-column-grid__expand-two">
+                    <img id="small-profile" src="/web-images/profile pic.png" alt="small-profile">
+                    <div class="profile-name">
+                        <p>Some legend guy</p>
+                        <p>@bubblebutcatlover</p>
+                    </div>
+                    <img class="dots" src="/web-images/3dots.png" alt="dots">
+
+                </div>
+            </div>
+        </header>
+        <main class="main three-row-grid__main">
```

Comment:
You can eliminate the need for a CSS `.main` class and simply select the `main` tag?

`.main { .... } ` -> `main { ... }`

In this way since the section tag is already called main, the CSS would read in the same way.